Add Panorays Findings API integration (#43630)
demisto/content: Packs/Panorays/README.md

# Panorays

> This pack provides integration with the **Panorays** platform to monitor and manage internal security findings and posture.

Reviewer: ```suggestion
This pack provides an integration with the **Panorays** platform to monitor and manage internal security findings and posture.
```

Author: Fixed and commited

## What does this pack do?

> This pack allows you to integrate Panorays' security posture data into Cortex XSOAR to automate the management of your organization's internal findings.

Reviewer: ```suggestion
This pack enables you to integrate Panorays security posture data into Cortex XSOAR to automate the management of your organization's internal findings.
```

> * **Automated Incident Ingestion:** Automatically fetch and create incidents in XSOAR based on new internal security findings identified by Panorays for your organization.

Reviewer: ```suggestion
* **Automated incident ingestion:** Automatically fetch and create incidents in Cortex XSOAR based on  internal security findings identified by Panorays for your organization.
```

> * **Internal Visibility:** Retrieve detailed lists of self-assessment findings, including severity, category, and affected assets, directly within the XSOAR war room.

Reviewer: ```suggestion
* **Internal visibility:** Retrieve detailed lists of self-assessment findings, including severity, category, and affected assets, directly within the Cortex XSOAR War Room.
```

> * **Posture Management:** Accelerate your response to internal security gaps by centralizing Panorays findings alongside your other security tools.

Reviewer: ```suggestion
* **Posture management:** Accelerate your response to internal security gaps by centralizing Panorays findings alongside your other security tools.
```
